src/Types/BlockHash.cs

```diff
@@ -27,6 +28,12 @@ private BlockHash(string blockHashAsBase16String)
     private BlockHash(byte[] blockHashAsBytes)
         : base(blockHashAsBytes) { }
 
+    /// <summary>
+    /// Creates an instance from a byte string object given from generated code. Only internal use.
+    /// </summary>
+    /// <param name="byteString">A block hash represented as a immutable array.</param>
+    internal static BlockHash From(ByteString byteString) => new(byteString.ToByteArray());
```

Comment:
Shouldn't this be checking length? Or be stated as a precondition that it is unsafe.

Reply:
This is done in the parent constructor